automatic/inkscape/tools/chocolateyInstall.ps1

```diff
@@ -49,7 +49,6 @@ Get-ChildItem $toolsPath\*.msi | ForEach-Object { Remove-Item $_ -ea 0; if (Test
 $packageName = $packageArgs.packageName
 $installLocation = Get-AppInstallLocation $packageArgs['softwareName']
 if ($installLocation) {
-  Install-BinFile 'inkscape' $installLocation\bin\inkscape.exe
```

Comment:
If this will be added, the disabling of the inkscape shim must be behind a package parameters flag that the user passes in.

ie, by default a shim gets created, but a user can decide to disable this generation if they want.